content/en/blog/2023/sunsetting-opencensus.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Sunsetting OpenCensus
+linkTitle: Sunsetting OpenCensus
+date: 2023-05-01
+author: '[Aaron Abbott](https://github.com/aabmass) (Google)'
+---
+
+In 2019, we announced that OpenTracing and OpenCensus would be merging to form
+the OpenTelemetry project. From the start, we considered OpenTelemetry
+[to be the next major version of both OpenTracing and OpenCensus](https://www.cncf.io/blog/2019/05/21/a-brief-history-of-opentelemetry-so-far/).
+
+We are excited to announce that OpenTelemetry has reached feature parity with
+OpenCensus in C++, C#, Go, Java, JavaScript, PHP and Python.
```

Comment:
Change "C#" to ".NET", this is also aligned with:

* https://github.com/census-instrumentation/opencensus-csharp#opencensus-net-sdk
* https://github.com/open-telemetry/opentelemetry-dotnet#opentelemetry-net